src/rdkafka.h

```diff
@@ -5384,6 +5384,8 @@ typedef int rd_kafka_event_type_t;
 #define RD_KAFKA_EVENT_DESCRIBETOPICS_RESULT 0x100000
 /** DescribeCluster_result_t */
 #define RD_KAFKA_EVENT_DESCRIBECLUSTER_RESULT 0x200000
+/** ListOffsets_result_t */
+#define RD_KAFKA_EVENT_LISTOFFSETS_RESULT 0x400000
```

Comment:
Add `RD_KAFKA_EVENT_LISTOFFSETS_RESULT` to the documentation of `rd_kafka_event_opaque`